scripts/release-test.sh

```
@@ -37,9 +38,54 @@ dev="$(
+  --app-server-commit "$commit" \
+  --app-server-built-at 2026-08-15T12:34:56Z \
+  --app-server-reused-from "" \
+  --macos-fingerprint "$mac_fingerprint" \
+  --macos-version 2026.8.14-dev.t221030+deadbeef \
+  --macos-commit "$commit" \
+  --macos-built-at 2026-08-14T22:10:30Z \
+  --macos-reused-from dev \
+  --output "$manifest"
+jq -e '.schemaVersion == 1 and .release.tag == "dev" and .components.macos.reusedFrom == "dev"' "$manifest" >/dev/null
+
+selection="$($SCRIPT_DIR/select-release-components.sh "$app_fingerprint" "$mac_fingerprint" "$manifest")"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)